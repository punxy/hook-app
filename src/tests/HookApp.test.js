import React from 'react';
import { shallow } from "enzyme";
import '@testing-library/jest-dom';
import { HookApp } from '../HookApp';


describe('Test in HookApp component', () => {
    test('should be render correctly', () => {
        const wrapper = shallow( <HookApp /> );
        expect(wrapper).toMatchSnapshot();
    });
});