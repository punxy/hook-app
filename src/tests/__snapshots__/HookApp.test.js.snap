// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test in HookApp component should be render correctly 1`] = `
<div>
  <h1>
    Hola Mundo
  </h1>
</div>
`;
